### PR TITLE
Bugfix/ changed of filedinoneyear values from True/False to Yes/No/Unknown

### PR DESCRIPTION
**Pull request by schandrupatla**
## Description

Worked on changing values of "filed_in_one_year" column  :
-True /False to Yes/No/Unknown
Changed the label on the web page. cases/"Filed within Year" to "Filed within One Year"

see the corresponding branch change link on the backend:
[backend branch changes](https://github.com/Lambda-School-Labs/human-rights-first-asylum-be-a/pull/170)

ticket link on Trello board:[ticket on trello](https://trello.com/c/j0KRCnE3)
Contributors: @jenna-anderson 
video link: [PR Video](https://www.loom.com/share/9bfc8331d7d04f72b4bffe426fc6f8d9)

Fixes # (issue)
-Fixed the values showing on the screen for column "Filed within one year" to show Yes/No/Unknown
-Changed the label of column "Filed within Year" to "Filed within One Year"

## Type of change
Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] This change requires a documentation update

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have removed unnecessary comments/console logs from my code
- [x] I have made corresponding changes to the documentation if necessary (optional)
- [x] My changes generate no new warnings
- [x] I have checked my code and corrected any misspellings
- [x] No duplicate code left within changed files
- [x] Size of pull request kept to a minimum
- [x] Pull request description clearly describes changes made & motivations for said changes

### CODE REVIEW

File: src/components/pages/Cases/CaseTable.js [hunk 1]
```diff
@@ -318,7 +318,7 @@ export default function CaseTable(props) {
       ...getColumnSearchProps('case_origin_state'),
     },
     {
-      title: 'Filed within Year',
+      title: 'Filed within One Year',
       dataIndex: 'filed_within_year',
       key: 'filed_within_year',
```
Comment (ZacharyCooremans) [inline src/components/pages/Cases/CaseTable.js:323]: I know we had a PR that hasn't been merged yet that fixes the naming of variables, but should 322, 323, and 324 have been changed?

**Review by ZacharyCooremans — APPROVED** [review]
I think the changes were very simple and easy to understand. If we need to change 322, 323, and 324 to prevent merge conflicts that might be a good idea. If we don't then good PR and changes.

File: src/components/pages/Cases/CaseTable.js (hunk 1 above)
Comment (AD9018) [reply src/components/pages/Cases/CaseTable.js:323]: Looks good to me, seems like a simple change but would check filed_within_year.

**Review by AD9018 — APPROVED** [review]
Looks good to me, seems like a simple change but would check filed_within_year.

File: src/components/pages/CaseOverview/CaseOverview.js [hunk 2]
```diff
@@ -72,10 +72,7 @@ const CaseOverview = props => {
                 {caseData.initial_or_appellate ? 'True' : 'False'}
               </p>
               <p>Case Origin: {caseData.case_origin}</p>
-              <p>
-                Filed 1 Year:{' '}
-                {caseData.filed_within_one_year ? 'True' : 'False'}
-              </p>
+              <p>Filed 1 Year: {caseData.filed_within_one_year}</p>
```
Comment (joshuascan) [inline src/components/pages/CaseOverview/CaseOverview.js:75]: The paragraph text on line 75 should probably be changed to "Filed Within One Year" for consistency, or at least changed to "Filed Within 1 Year".

File: src/components/pages/Cases/CaseTable.js (hunk 1 above)
Comment (joshuascan) [reply src/components/pages/Cases/CaseTable.js:323]: I actually don't think those were changed in that pull request as we were searching for and revising "filed_in_one_year" and not "within". I'm not sure if these are something different though or if maybe we overlooked them. Those were also changed to "check_for_one_year" so I'm not sure if that would need to be applied here as well.

**Review by joshuascan — CHANGES_REQUESTED** [review]
I think that line 75 in CaseOverview should probably be revised per my note for consistency and readability, but I can approve if you guys feel it is better to leave it alone.

**Review by paulstgermain — APPROVED** [review]
In an effort to keep development flowing, approving this and leaving feedback tomorrow afternoon. :)

**Pull request merged**